### PR TITLE
feat: [PIE-8831]: add reload from cache option for run pipeline form …

### PR DESCRIPTION
**Pull request by vikrant-harness**
…and re-run pipeline form

### Summary

- Added reload from git option on run pipeline form / re-run pipeline form 
- Updated the re-run icon on the studio as well

#### Screenshots

https://user-images.githubusercontent.com/106532291/228513214-13e9dc0d-7de8-4d1e-94b1-0a8c14f8628a.mov

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/70-pipeline/components/PipelineStudio/PipelineCanvas/EntityCachedCopy/EntityCachedCopy.tsx [hunk 1]
```diff
@@ -57,10 +57,19 @@ function EntityCachedCopyInner(
   }
 
   const tooltipContent = (
-    <>
+    <div className={css.popover}>
```
Comment (AshwinBhatkal-Harness) [inline src/modules/70-pipeline/components/PipelineStudio/PipelineCanvas/EntityCachedCopy/EntityCachedCopy.tsx:60]: Is classname required here? Think all of them can be handled with props

**Review by AshwinBhatkal-Harness — DISMISSED** [review]

**Review by vkbansal-harness — APPROVED** [review]

File: src/modules/70-pipeline/components/PipelineStudio/PipelineCanvas/EntityCachedCopy/EntityCachedCopy.tsx (hunk 1 above)
Comment (vikrant-harness) [reply src/modules/70-pipeline/components/PipelineStudio/PipelineCanvas/EntityCachedCopy/EntityCachedCopy.tsx:60]: Will check this

**Comment by vikrant-harness** [comment]
retrigger typecheck

**Review by AshwinBhatkal-Harness — APPROVED** [review]

**Comment by vikrant-harness** [comment]
The coverage check is failing due to low coverage in the existing components.

**Pull request merged**